src/services/auth.js

```diff
@@ -34,7 +38,7 @@ const authService = {
       throw createError(401, USER_NOT_FOUND)
     }
 
-    const checkedPassword = (password === user.password) || isFromGoogle
+    const checkedPassword = password === user.password || isFromGoogle
```

Comment:
It's easier to check `isFromGoogle ` first
```suggestion
    const checkedPassword = isFromGoogle || password === user.password
```